CONTRIBUTING.md

```diff
@@ -0,0 +1,257 @@
+# Contributing guidelines
+
+**Want to contribute? Great!** 
+We try to make it easy, and all contributions, even the smaller ones, are more than welcome.
+This includes bug reports, fixes, documentation, examples... 
+But first, read this page.
+
+## Reporting an issue
+
+This project uses GitHub issues to manage the issues. Open an issue directly in GitHub.
+
+If you believe you found a bug, and it's likely possible, please indicate a way to reproduce it, what you are seeing and what you would expect to see.
+Don't forget to indicate your Quarkus, Java, Maven/Gradle and GraalVM version. 
+
+### Code reviews
+
+All submissions, including submissions by project members, need to be reviewed before being merged.
+
+## Setup
+
+If you have not done so on this machine, you need to:
+ 
+* Install Git and configure your GitHub access
+* Install Java SDK (OpenJDK recommended)
+* Install [GraalVM](https://quarkus.io/guides/building-native-image)
+* Install platform C developer tools:
+    * Linux
+        * Make sure headers are available on your system (you'll hit 'Basic header file missing (<zlib.h>)' error if they aren't).
```

Comment:
Where do "headers" come from in this case?

Reply:
In linux distros the headers are usually installed with the distro's package manager. But I agree that the wording could be improved a bit here, maybe `Make sure header files are installed on your system`?